src/main/webapp/app/exercises/programming/shared/lifecycle/programming-exercise-lifecycle.component.ts

```diff
@@ -132,9 +132,9 @@ export class ProgrammingExerciseLifecycleComponent implements AfterViewInit, OnD
         }
     }
 
-    toggleManualFeedbackRequests() {
-        this.exercise.allowManualFeedbackRequests = !this.exercise.allowManualFeedbackRequests;
-        if (this.exercise.allowManualFeedbackRequests) {
+    toggleFeedbackRequests() {
```

Comment:
`toggleManualFeedbackRequests` was renamed to `allowFeedbackRequests` on `develop` - there was a conflict because the method above `toggleSetTestCaseVisibilityToAfterDueDate` was added in this PR (= change in the same line)